### PR TITLE
[ONNX] Split ONNX tests into smaller files in test/onnx subdirectory

### PR DESCRIPTION
**Pull request by postrational**

### CODE REVIEW

File: test/onnx/onnx_import.in.cpp [hunk 1]
```diff
@@ -0,0 +1,1412 @@
...
+
+    std::size_t size = function->get_output_size();
+    for (std::size_t i{0}; i < size; ++i)
+    {
+        std::shared_ptr<Node> node = function->get_output_op(i);
+        EXPECT_EQ(node->get_friendly_name(), "output_" + std::to_string(i + 1));
+    }
+}
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_add_abc)
```
Comment (tsocha) [inline test/onnx/onnx_import.in.cpp]: Why add_abc, binary_add_abc and add_abc_initializers tests are in the core section but addmul_abc is in operators section?

File: test/onnx/onnx_import.in.cpp [hunk 2]
```diff
@@ -0,0 +1,1412 @@
...
+            {{{{2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}},
+              {{2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}},
+              {{2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}, {2, 3, 4, 5, 6}}}})
+            .get_vector()};
+
+    Outputs outputs{execute(function, inputs, "${BACKEND_NAME}")};
+    EXPECT_TRUE(test::all_close_f(expected_output.front(), outputs.front()));
+}
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_reduce_log_sum)
```
Comment (tsocha) [inline test/onnx/onnx_import.in.cpp]: Maybe reduce tests should be in another file?

File: test/onnx/onnx_import.in.cpp (hunk 1 above)
Comment (postrational) [reply test/onnx/onnx_import.in.cpp]: These tests were actually designed to test core functionality, so I grouped them there.

File: test/onnx/onnx_import.in.cpp (hunk 2 above)
Comment (postrational) [reply test/onnx/onnx_import.in.cpp]: Agreed, but I think we can do this in the future.

**Pull request merged**